server/services/EventHandlers.js

```diff
@@ -0,0 +1,32 @@
+const Session = require('../database/Session');
```

Comment:
We may need some event handlers in this project.  We could leave this file and the function `registerEventHandler` and remove the `handleVerifyCaptureProcessed`. This specific function is a handler for events raised in admin panel to be handled in the treetracker-field-data service.

Reply:
Done!